From 12ef06df3491a62b199723caabf320d50528a6bf Mon Sep 17 00:00:00 2001
From: Huang Rui <ray.huang@amd.com>
Date: Wed, 13 Feb 2019 10:44:50 +0800
Subject: [PATCH 2264/2940] drm/amd/powerplay: update smu11 driver if header
 for navi10 (v2)

This patch updates smu11 driver if header for navi10 to match 42.09.00 smu
firmware.

v2: clean up comments

Signed-off-by: Huang Rui <ray.huang@amd.com>
Reviewed-by: Hawking Zhang <Hawking.Zhang@amd.com>
Signed-off-by: Alex Deucher <alexander.deucher@amd.com>
---
 .../gpu/drm/amd/powerplay/inc/smu_11_0_driver_if.h    | 11 ++++++++++-
 1 file changed, 10 insertions(+), 1 deletion(-)

diff --git a/drivers/gpu/drm/amd/powerplay/inc/smu_11_0_driver_if.h b/drivers/gpu/drm/amd/powerplay/inc/smu_11_0_driver_if.h
index b98cb005a46c..a53547fa8980 100644
--- a/drivers/gpu/drm/amd/powerplay/inc/smu_11_0_driver_if.h
+++ b/drivers/gpu/drm/amd/powerplay/inc/smu_11_0_driver_if.h
@@ -4,7 +4,7 @@
 // *** IMPORTANT ***
 // SMU TEAM: Always increment the interface version if 
 // any structure is changed in this file
-#define SMU11_DRIVER_IF_VERSION 0x2D
+#define SMU11_DRIVER_IF_VERSION 0x2E
 
 #define PPTABLE_NV10_SMU_VERSION 8
 
@@ -297,6 +297,15 @@ typedef struct {
   
 } SwI2cRequest_t; // SW I2C Request Table
 
+//D3HOT sequences
+typedef enum {
+  BACO_SEQUENCE,
+  MSR_SEQUENCE,
+  BAMACO_SEQUENCE,
+  ULPS_SEQUENCE,
+  D3HOT_SEQUENCE_COUNT,
+}D3HOTSequence_e;
+
 //THis is aligned with RSMU PGFSM Register Mapping
 typedef enum {
   PG_DYNAMIC_MODE = 0,
-- 
2.17.1

